contracts/src/ModuleCRC21.sol

```diff
@@ -0,0 +1,34 @@
+pragma solidity ^0.6.8;
+
+import "./ModuleCRC20.sol";
+
+contract ModuleCRC21 is ModuleCRC20  {
+
+    event __CronosSendToEthereum(address sender, address recipient, uint256 amount, uint256 bridge_fee);
+    event __CronosCancelSendToEthereum(address sender, uint256 id);
+
+    constructor(string memory denom_, uint8 decimals_) ModuleCRC20(denom_, decimals_) public {
+        decimals = decimals_;
+        denom = denom_;
+    }
+
+    // make unsafe_burn internal
+    function unsafe_burn_v2(address addr, uint amount) internal {
+        // Deduct user's balance without approval
+        require(balanceOf[addr] >= amount, "ds-token-insufficient-balance");
+        balanceOf[addr] = sub(balanceOf[addr], amount);
+        totalSupply = sub(totalSupply, amount);
+        emit Burn(addr, amount);
+    }
+
+    // send to ethereum through gravity bridge
```

Comment:
Would it be better to make it more generic such as "send_to_chain(....uint chainId)" considering that multi bridge is on the roadmap?

Reply:
sounds good, and we don't have to use the `_v2` suffix.